core/plugins/org.fusesource.ide.camel.model.service.impl/META-INF/MANIFEST.MF

```diff
@@ -12,17 +12,18 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.11.0",
  org.jboss.tools.foundation.core;bundle-version="1.2.0",
  org.jboss.tools.foundation.ui;bundle-version="1.2.0",
  org.fusesource.ide.foundation.core;bundle-version="10.0.0",
- org.fusesource.ide.camel.model.service.core;bundle-version="10.0.0"
+ org.fusesource.ide.camel.model.service.core;bundle-version="10.0.0",
+ org.apache.commons.httpclient;bundle-version="3.1.0"
```

Comment:
is that lib a new requirement?

Reply:
I created https://github.com/jbosstools/jbosstools-fuse/pull/1087 to remove it

Reply:
finally this wasn't added by error, now that tests are failing on the other PR, i remember now that it was needed